efel/cppcore/LibV1.cpp

```diff
@@ -571,9 +573,15 @@ int LibV1::AHP_depth_abs_slow(mapStr2intVec& IntFeatureData,
         "AHP_slow_time.\n";
     return -1;
   }
+  // time after the spike in ms after which to start searching for minimum
+  vector<double> sahp_start;
+  retval = getVec(DoubleFeatureData, StringData, "sahp_start", sahp_start);
+  if (retval < 0){
+    sahp_start.push_back(5);
```

Comment:
I think it would be better if the default value was defined here:
https://github.com/BlueBrain/eFEL/blob/a9281517f1e05a7596abb9773f9807ba10432ddd/efel/api.py#L101